src/plugins/ucx/ucx_backend.cpp

```diff
@@ -653,7 +653,7 @@ nixl_status_t nixlUcxEngine::endConn(const std::string &remote_agent) {
 }
 
 nixl_status_t nixlUcxEngine::getConnInfo(std::string &str) const {
-    str = nixlSerDes::_bytesToString(workerAddr.get(), workerSize);
+    str = nixlSerDes::_bytesToString(workerAddr.data(), workerAddr.size());
```

Comment:
seems it could be just `str = workerAddr` now
btw, maybe keep the original format of meta data to avoid extra string copies along the stack?